### PR TITLE
boards/multiboot: close doxy group

### PR DESCRIPTION
**Pull request by yogo1212**

### CODE REVIEW

**Comment by A-Paul** [comment]
Hey @yogo1212, thanks for finding this issue and fixing it.

File: boards/x86-multiboot-common/include/multiboot.h [hunk 1]
```diff
@@ -235,3 +235,5 @@ typedef struct multiboot_mod_list multiboot_module_t;
 #endif
 
 #endif  /** ! MULTIBOOT_H_ */
+
```
Comment (A-Paul) [inline boards/x86-multiboot-common/include/multiboot.h]: Please remove empty line.

**Comment by yogo1212** [comment]
@A-Paul ping :-)
i've allowed myself the brazenness of amending the existing commit because of the low number of changes

**Comment by haukepetersen** [comment]
ACK once the CI is green

**Comment by Kijewski** [comment]
ACK

**Comment by authmillenon** [comment]
Needs backport.

**Comment by OlegHahm** [comment]
Not so sure, it's only a doxygen fix.

**Comment by authmillenon** [comment]
Would prefer to have one.

**Pull request merged**